src/librustc_typeck/check_unused.rs

```diff
@@ -75,7 +75,7 @@ impl<'a, 'tcx> CheckVisitor<'a, 'tcx> {
         let msg = if let Ok(snippet) = self.tcx.sess.source_map().span_to_snippet(span) {
             format!("unused import: `{}`", snippet)
         } else {
-            "unused import".to_string()
+            "unused import".to_owned()
```

Comment:
I also prefer `.to_owned` over `.to_string` for literals (the former indicates that we're getting ownership of something which is already a string in some sense of the word, whereas the latter could be anything `Display`-able), but it's possible that we're in the minority insofar as [The Book recommends](https://doc.rust-lang.org/nightly/book/second-edition/ch08-02-strings.html#creating-a-new-string) `.to_string` or `String::from`.

Reply:
I think The Book just recommends the solution that is simplest to grasp and remember; `to_string` calls `String::from` which, in turn, calls `to_owned()`, so everything boils down to `to_owned` anyways.